docs/kb/semgrep-supply-chain/sss-python-lockfiles.md

```diff
@@ -0,0 +1,294 @@
+---
+description: Generation of Python lock files to run SSC scans successfully.
```

Comment:
```suggestion
description: Generate various Python lock files to run Semgrep Supply Chain scans successfully.
```
We have to spell out SSC in case our users don't know what it is :)